students/jiyeon/users/models.py

```diff
@@ -0,0 +1,14 @@
+from django.db import models
+from django.db.models.deletion import CASCADE
+
+
+class User(models.Model):
+    name = models.CharField(max_length=30)
+    email = models.CharField(max_length=200, unique=True)
+    password = models.CharField(max_length=30)
+    phone_number = models.CharField(max_length=30)
```

Comment:
1. 컨벤션 맞춰주세요~
2. password property는 나중에 해시(암호화)된 비밀번호 길이를 고려해서 넉넉하게 200 이상으로 설정해주세요.
```suggestion
    name         = models.CharField(max_length=30)
    email        = models.CharField(max_length=200, unique=True)
    password     = models.CharField(max_length=30)
    phone_number = models.CharField(max_length=30)
```